### PR TITLE
Support ESF-RAW message [STAR-951]

### PR DESCRIPTION
**Pull request by yizow**

### CODE REVIEW

File: c/test/check_ubx.c [hunk 1]
```diff
@@ -339,6 +349,33 @@ START_TEST(test_ubx_rxm_sfrbx) {
...
+  uint8_t buff[1024];
+  memset(buff, 0, 1024);
+  ubx_encode_esf_raw(&msg, buff);
+
+  ubx_esf_raw msg_esf_raw_out;
+  int8_t ret = ubx_decode_esf_raw(buff, &msg_esf_raw_out);
+  ck_assert_int_eq(RC_OK, ret);
+  msg_esf_raw_equals(&msg, &msg_esf_raw_out);
+
+  return;
```
Comment (jbangelo) [inline c/test/check_ubx.c]: Is the trailing return needed by the testing framework? It looks like you've used it elsewhere in the file.

File: c/test/check_ubx.c [hunk 2]
```diff
@@ -339,6 +349,33 @@ START_TEST(test_ubx_rxm_sfrbx) {
...
+  msg.length = 4 + 8 * 4;
+
+  for (int i = 0; i < 4; i++) {
+    msg.data[i] = 1 << i;
+    msg.sensor_time_tag[i] = i;
+  }
+
+  uint8_t buff[1024];
+  memset(buff, 0, 1024);
+  ubx_encode_esf_raw(&msg, buff);
```
Comment (jbangelo) [inline c/test/check_ubx.c]: Wouldn't be bad to make sure the return value here is correct as well.

File: c/test/check_ubx.c (hunk 1 above)
Comment (yizow) [reply c/test/check_ubx.c]: Not needed; removed.

**Comment by yizow** [comment]
Haven't tested with real data yet. Logic for parsing semantics of message currently resides in gnss-converters.

**Pull request merged**